widgetssdk/src/main/java/com/glia/widgets/UiTheme.kt

```diff
@@ -138,6 +138,24 @@ data class UiTheme(
     @ColorRes
     val gliaChatHeaderExitQueueButtonTintColor: Int? = null,
 
+    /**
+     * TODO
```

Comment:
Also to double-check - are you going to add docs later?

Reply:
Fixed: 345ce4c